docs/process-based-load-script.md

```diff
@@ -0,0 +1,114 @@
+# Processes based load script
+
+* [Requirements](#requirements)
+* [Parameters description](#parameters-description)
+* [Examples](#examples)
+
+## Requirements
+
+* native libs
+    * libindy >= 1.4.0
+
+* python lib
+    * python3-indy >= 1.4.0
+    * libnacl == 1.6.1
+    
+## Parameters description
+
+'-c', '--clients' : Number of independent processes to run.
+Each process will use its own pool handle and wallet and will send requested number of specified pool txns.
+If less or equal than 0 value provided the number of processes will be equal to number available CPUs.
+Default value is 0.
+
+'-g', '--genesis' : Path to file with genesis txns for the pool to connect to.
+
+'-s', '--seed' : Seed that will be used to generate submitter did.
+
+'-w', '--wallet_key' : Key to access encrypted wallet
+
+'-n', '--num' : Number or txns in one batch. If timeout param omitted only one batch will be sent.
+
+'-t', '--timeout' : Timeout between batches. If omitted only one batch will be sent.
```

Comment:
May be it make sense to add default values of parameters?

Reply:
fixed